src/main/java/seedu/duke/exercisegenerator/GenerateExercise.java

```diff
@@ -21,28 +21,24 @@ public class GenerateExercise {
     private static final String EXPERT = "expert";
     private static final String OPEN_BRACE = "[";
     private static final String CLOSE_BRACE = "]";
-    public ArrayList<ExerciseData> exerciseData;
+    private final ArrayList<ExerciseData> exerciseDataList;
+
+    private ArrayList<ExerciseData> filteredExerciseList;
 
     public GenerateExercise() {
         ParseData parseData = new ParseData();
-        exerciseData = parseData.getExercises();
-    }
-
-    public ArrayList<ExerciseData> generateRandomSet(int count) {
+        exerciseDataList = parseData.getExercises();
```

Comment:
Good removal of quickstart command, as it serves the exact same function as the generate command.